tools/serve/serve.py

```diff
@@ -454,6 +516,10 @@ def __init__(self):
                           ("*", "/results/", handlers.ErrorHandler(404))]
 
         self.extra = []
+        self.inject_script_data = None
+        if inject_script is not None:
+            with open(inject_script, 'r') as f:
```

Comment:
Suggesting binary to match previous comment/question:

```suggestion
            with open(inject_script, 'rb') as f:
```

Reply:
Done.